khi_robot_control/src/khi_robot_krnx_driver.cpp

```diff
@@ -829,30 +897,40 @@ bool KhiRobotKrnxDriver::makeRtcParam( const int cont_no, const std::string name
             fprintf( fp, ".END\n" );
         }
         fclose( fp );
+    }
+    else
+    {
+        return false;
+    }
 
-        /* HOME position */
-        for ( int ano = 0; ano < robot_info[cont_no].arm_num; ano++ )
-        {
-            /* AS */
-            snprintf( cmd_buf, sizeof(cmd_buf), "HERE/N %d: #rtchome%d", ano+1, ano+1 );
-            return_code = krnx_ExecMon( cont_no, cmd_buf, msg_buf, sizeof(msg_buf), &error_code );
+    return_code = krnx_Load( cont_no, file_path );
+    unlink( file_path );
+    if ( !retKrnxRes( cont_no, "krnx_Load", return_code ) ) { return false; }
+
+    return true;
+}
+
+bool KhiRobotKrnxDriver::syncRtcPos( const int cont_no, JointData *joint )
```

Comment:
`JointData *joint` is not used in this func.

Reply:
Yes, I removed argument `joint`.